src/components/dialogs/parameters/load-flow-parameters.jsx

```diff
@@ -582,108 +589,132 @@ export const LoadFlowParameters = ({ parametersBackend }) => {
     // we must keep the line of the simulator selection visible during scrolling
     // only specifics parameters are dependents of simulator type
     return (
-        <>
-            <Grid sx={{ height: '100%' }} xl={6}>
-                <Grid
-                    container
-                    spacing={1}
+        <Grid item sx={{ height: '100%' }} xl={8} lg={10} md={12}>
```

Comment:
A grid item should be wrapped by a grid container, however at the place which uses this component, the grid is an item too, see https://github.com/gridsuite/gridstudy-app/blob/main/src/components/parameters-tabs.tsx#L493

In my opinion, this should be a container gird

Reply:
This is already in a container, see parameters-tabs.tsx line 376

Reply:
Note : we cannot put a container here, it breaks everything.
These whole pages have to be refactored someday because they are all over the place, but for now, this works.